nph.nimble

```diff
@@ -12,7 +12,7 @@ bin           = @["nph"]
 # TODO https://github.com/nim-lang/nimble/issues/1166
 # Using exact version here and adding path manually :facepalm:
 # run `nimble setup -l` to hopefully make it work
-requires "nim >= 2.0.0 & <= 2.0.2",
+requires "nim >= 2.0.0 & <= 2.0.4",
```

Comment:
Perhaps this could be updated to support any 2.0.X patch version without having to make a code change?

Reply:
I guess we can yolo this, hoping that no parser updates will be included in patch versions..

this needs a proper solution at some point before 1.0 though, where we can compare the formatted code against arbitrary nim versions

Reply:
aka let's make it accept any patch version but document that the parser used by the formatter and the parser used by the verifier might disagree, with a TODO noting that nph should learn how to check formatting with any nim version

Reply:
Okay, sounds good. I've updated my changes as suggested.